ideas/71-low-traffic.md

```diff
@@ -0,0 +1,75 @@
+## Preamble
+    Idea: <to be assigned>
+    Title: Low traffic consumption(technical)
+    Status: Draft
+    Created: 2018-01-25
+
+## Summary
+1) Issues related to statusgo network overhead
+
+## Swarm Participants
+- Lead Contributor: @JekaMas  (40h/w)
+- Testing & Evaluation: @annadanchenko (?)
+- Go Contributor: @b00ris (40h/w)
+- Go Contributor: @PombeirP  (30h/w)
+- Go Contributor: @feuGeneA (30h/w)
+- Clojure Contributor: @yenda (20h/w)
+
+## Product Overview
+Measure network overhead for main `statusgo` usage scenarios.
+Identify and fix the bottlenecks.
+
+### Product Description
+Main use cases to track:
+1) node was started in background(sync was ended)
+1.1) node was started in background(sync was ended) + whisper
+2) node was started in background(sync is running)
+2) node was started in background(upstream)
+2.1) node was started in background(upstream) + whisper
+3) one chat 1-1.  sending messages(60rpm)
+4) 60 chats 1-1. receiving messages(10rpm per chat)
+5) 10 group chats(60 rpm per chat)
+6) 100 chats 1-1. 30 group chats. receiving historic messages for last day(100 messages per chat)
+
+
+Many topics problem https://github.com/status-im/status-react/issues/2964
+Whisper network overhead https://github.com/status-im/status-react/issues/2931
+
+### Requirements & Dependencies
+Part of https://github.com/status-im/ideas/issues/55
+Related to ultralight client
+
+## [Iterations](#iterations)
```

Comment:
Great list! For historical iterations, can we post a short update on them?

Is the one for 9 April still accurate or where are we? To me it seems like we are in middle of iteration 3, so probably want to adjust dates.

Is the new protocol thing going to be its own idea or solved within context of this swarm or what's the plan?